api/src/test/java/jakarta/data/repository/PageableTest.java

```diff
@@ -27,6 +27,7 @@
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.assertj.core.api.SoftAssertions.assertSoftly;
 
+//@DisplayNameGeneration(ReplaceCamelCaseAndUnderscore.class)
```

Comment:
Of course , I will add this annotation on other test classes as well. I just wanted to solve the current issues before moving forward.

Reply:
Changes look good 👍🏼